server/config.ts

```diff
@@ -174,4 +174,9 @@ export default {
   analytics: {
     tagManagerContainerId: get('TAG_MANAGER_CONTAINER_ID', ''),
   },
+  rollout: {
+    restricted: get('RESTRICT_ROLLOUT', false),
+    probationAreas: ['N55'],
+    prisons: ['MDI', 'LEI'],
+  },
 }
```

Comment:
This is switched off by default. To switch on we setup the pilot areas and prisons here, and set an environment variable f `RESTRICT_ROLLOUT` to true. Tests temporarily switch this on to check functionality.